semgrep/semgrep/rule_schema.yaml

```diff
@@ -3,6 +3,7 @@ $schema: http://json-schema.org/draft-07/schema#
 definitions:
   patterns-content:
     type: array
+    title: "Return finding where all of the nested conditions are true"
```

Comment:
Nit: this one has quotes for `title` and the others don't, not sure if this is intentional